CHANGES.md

```diff
@@ -1,3 +1,77 @@
+## stable-2.6.0
+
+This release introduces experimental distributed tracing support, adds request
+and response headers to `linkerd tap`, dramatically improves the performance of
+the dashboard on large clusters, adds traffic split visualizations to the
+dashboard, adds a public Helm repo, and many more improvements!
+
+For more details, see the announcement blog post: TODO
+https://linkerd.io/2019/08/20/announcing-linkerd-2.6/
+
+To install this release, run: `curl https://run.linkerd.io/install | sh`
+
+**Upgrade notes**: Use the `linkerd upgrade` command to upgrade the control
+plane. This command ensures that all existing control plane's configuration and
+mTLS secrets are retained. For more details, please see the [upgrade
+instructions](https://linkerd.io/2/tasks/upgrade/).
+
+**Special thanks to**: @alenkacz, @arminbuerkle, @bmcstdio, @bourquep,
+@brianstorti, @kevtaylor, @KIVagant, @pierDipi, and @Pothulapati!
+
+**Full release notes**:
+
+* CLI
+  * Added a new `json` output option to the `linkerd tap` command, which exposes
+    request and response headers
+  * Added a public Helm repo - this release can be installed with: `helm repo add
+    linkerd https://helm.linkerd.io/stable && helm install linkerd/linkerd2`
+  * Added an `--address` flag to `linkerd dashboard`, allowing users to specify
+    a port-forwarding address (thanks @bmcstdio!)
+  * Added node selector constraints to Helm installation, so users can control
+    which nodes the control plane is deployed to (thanks @bmcstdio!)
+  * Added a `--cluster-domain` flag to the `linkerd install` command that allows
+    setting a custom cluster domain (thanks @arminbuerkle!)
+  * Added a `--disable-heartbeat` flag for `linkerd` `install | upgrade` commands
+  * Fixed the `linkerd endpoints` command to use the correct Destination API
```

Comment:
This was a change in controller and can be left out, as this was caused by separating out the destination component, which happened part of a previous edge release. @alpeb

Reply:
I see, thanks @Pothulapati!